.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,7 +7,10 @@
 ## Checklist
 _put `n/a` if item is not relevant to PR changes_
 
-- [ ] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
-- [ ] Have new automated tests been implemented?
-- [ ] Have new manual tests been written down?
-- [ ] Have any new strings been translated?
+- [ ] Has the CHANGELOG been updated?
+- [ ] Has the README been updated?
+- [ ] Has the RELEASE_GUIDELINES been updated?
+- [ ] Has the MIGRATION doc been updated for any major/minor breaking changes in the PR?
```

Comment:
I would make it more explicit on what we mean by updating the migration notes for minor changes.
```
[ ] Has the MIGRATION doc been updated for any MAJOR breaking changes?
[ ] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?

Reply:
Are these the only reasons we update `minor`? I think there might be more reasons that we do it hence I'm not sure listing them in the checklist. Maybe we could have some external document created with the points what is causing the bump of the `patch`, `minor` and `major` version. The link could be added to the checklist after 🤔

Reply:
Yes, it's the only reason. I think it would be good to have it in the checklist because it's quite uncommon to have migration notes for minor changes. We should still have a document in our wiki with the details of our migration process. And yes, if in the future we need to add more cases, we can refer to the wiki.

Reply:
Makes sense. Thank you for the explanation.

Reply:
Re: wiki, we already have it https://wiki.onfido.co.uk/pages/viewpage.action?spaceKey=ENG&title=SDK+Git+Workflow and there is a bit of explanation in the _Versioning_ section.

Reply:
Cool, we need to add a note about when to update the migration file and what to put in it. Can be done as a separate story. Can you delete the internal link, please? 🙏

Reply:
I left my suggestions on when to update the migration in the _Redefine SDK API process change - versioning, documentation_ google doc that I was sharing with the Team last week on Slack.